control/proto/gateway.proto

```diff
@@ -206,6 +209,13 @@ message add_host_req {
 	optional string dhchap_ctrlr_key = 5;
 }
 
+message change_host_keys_req {
+	string subsystem_nqn = 1;
+	string host_nqn = 2;
+	optional string dhchap_key = 3;
```

Comment:
Looks like dhcap_key is mandatory. Could we omit `optional` here?

Reply:
I usually do that not to harm compatibility but I guess as this request is new I can make it mandatory.

Reply:
guess assert dhcap_key and the if dhcap_key check could be removed in grpc.py then?

Reply:
I changed it back to how it was. You can use the command with no key. In this case a host which had a key before will now not have a key.